packages/cli/src/loaders/index.ts

```diff
@@ -109,5 +110,12 @@ export default function createBucketLoader(
       createFlatLoader(),
       createUnlocalizableLoader(),
     );
+    case 'xliff': return composeLoaders(
+      createTextFileLoader(bucketPathPattern),
+      // createPrettierLoader({ parser: 'xliff' }),
```

Comment:
i don't think prettier knows anything about xliff (it might know about xml though)

in any case, no reason to leave comments here